src/commands/account/get_public_key/mod.rs

```diff
@@ -0,0 +1,42 @@
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+mod from_keychain;
+#[cfg(feature = "ledger")]
+mod from_ledger;
+mod from_legacy_keychain;
+mod from_seed_phrase;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = crate::GlobalContext)]
+pub struct GetPublicKey {
+    #[interactive_clap(subcommand)]
+    get_public_key_mode: GetPublicKeyMode,
+}
+
+#[derive(Debug, Clone, EnumDiscriminants, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = crate::GlobalContext)]
+#[strum_discriminants(derive(EnumMessage, EnumIter))]
+/// Where do you want to get the public key from?
+pub enum GetPublicKeyMode {
+    #[cfg(feature = "ledger")]
+    #[strum_discriminants(strum(
+        message = "from-ledger                 - Get the public key stored on your Ledger Nano device"
+    ))]
+    /// Get the public key stored on your Ledger Nano device
```

Comment:
Not sure if these comments have any value as they are repeating text from the message above.

Reply:
`--help` uses these comments:

![Screenshot 2025-01-17 at 16 36 18](https://github.com/user-attachments/assets/46d4434b-9233-4b4d-bfc6-9674f0213818)

Reply:
Isn't it use strum macro above ?

Reply:
strum macro processes these messages with various functions

![Screenshot 2025-01-17 at 20 46 23](https://github.com/user-attachments/assets/29c256af-47a1-415b-b5b0-ad21c1b97e81)